app/js/components/LabResultsList.jsx

```diff
@@ -43,7 +43,11 @@ const Cell = ({
   if (columnName === 'RESULT') {
     return (
       <div className="table_cell result">
-        {getConceptShortName(value.value, locale)}
+        <ConceptDisplay
+          conceptUUID={value.concept.uuid}
+          type="result"
+          value={value.value ? getConceptShortName(value.value, locale) : null}
+        />
```

Comment:
This doesn't seem right.  And it looks like I missed the error in the last PR we worked on as well that is what broke this,

There are a couple keys here:
* The top-level "value" variable is poorly named which makes things confusing, it's really an obs. We should have named that variable "obs".
* The obs value could be a concept or value (for numeric obs)

The ConceptDisplay tag takes the uuid of the question concept, and the a value that is either a string or a numeric.  

So I think the logic should be more like this pseudo code:

// to make things clearer
let obs = value

// conceptUUID to past to Concept Display (uuid of the "question" concept)
conceptUUID = obs.concept.uuid

// for the value
if (obs.value is a concept) then getConceptShortName(obs.value, locale)
else if (obs.value is a numeric) then obs.value